src/pdf.image_decoders.js

```diff
@@ -20,7 +20,7 @@ import {
 } from "./shared/util.js";
 import { Jbig2Image } from "./core/jbig2.js";
 import { JpegImage } from "./core/jpg.js";
-import { JpxImage } from "./core/jpx.js";
```

Comment:
Removing `JpxImage` from the exports makes this a breaking change because `JpxImage` is exposed publicly. Note how for example the fuzzer relies on it at https://github.com/mozilla/pdf.js/blob/master/test/fuzz/jpx_image.fuzz.js#L2 but also the API test at https://github.com/mozilla/pdf.js/blob/7bd7c6ca5a68e1f35bfd586cf2e4c14890ac920d/test/unit/pdf.image_decoders_spec.js#L23. If we do this, we might need to bump the major version (but if we do it might make sense to also replace the other custom image decoders with OpenJPEG first where possible to do it all in one go; see the other comment about that below).